platform-includes/configuration/event-scrubber/python.mdx

```diff
@@ -25,3 +25,15 @@ sentry_sdk.init(
     event_scrubber=EventScrubber(denylist=denylist),
 )
 ```
+
+The scrubber will check parts of the event that are most likely to contain data from the denylist, like user and request data. It will, however, not search these recursively by default for performance reasons. If you want the scrubber to be recursive, initialize it with `recursive=True`.
```

Comment:
```suggestion
The scrubber will check parts of the event that are most likely to contain data from the denylist, like user and request data. However, by default, it will not search these recursively for performance reasons. If you want the scrubber to be recursive, initialize it with `recursive=True`.
```